hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionRemoteProcedureBase.java

```diff
@@ -365,4 +363,17 @@ protected void deserializeStateData(ProcedureStateSerializer serializer) throws
   protected void afterReplay(MasterProcedureEnv env) {
     getParent(env).attachRemoteProc(this);
   }
+
+  @Override public String getProcName() {
```

Comment:
The difference of ProcName and StringClassDetails is? The targetServer should add to toStringClassDetails?

Reply:
proc name should be short. stringclassdetails should be all parameters. Sometimes the name and the stringclassdetails are the same if there is nothing much to show for a procedure.

In this case, I think target server is a fundamental attribute of RegionRemoteProcedure so thought it should be in the proc name. Could move it to the string details.....Let me do that.